cnf-tests/Dockerfile

```diff
@@ -4,7 +4,7 @@ COPY . .
 RUN make test-bin
 RUN git rev-list -1 HEAD > ./cnf-tests/bin/cnftests-sha.txt
 
-FROM quay.io/openshift/origin-oc-rpms:4.6 AS oc
+FROM quay.io/openshift/origin-oc-rpms:4.7 AS oc
```

Comment:
looks good, but did you check we have these images available already? (I'd not just assume that)

Reply:
Let's wait for CI to tell.
I hope the mirroring jobs are already in place

Reply:
works for me